great_expectations/data_context/store/expectations_store.py

```diff
@@ -171,6 +171,9 @@ def update_expectation(
                 break
 
         self.update(key=suite_identifier, value=fetched_suite)
+
+        # Ensure that state of local suite is in sync with cloud backend
+        suite.expectation_configurations = fetched_suite.expectation_configurations
```

Comment:
Need to add this to get `expectation.save()` to work as expected

Reply:
What issue were you seeing before/what wasn't expected? 
It may be strange that saving on expectation syncs the whole suite. For example, if you've modified other expectations in the suite but haven't saved them it, those changes will get overwritten here.

Reply:
`Expectation.save()` did not update the suite at all - I can modify this logic so we only update the specific expectation within the suite instead of everything